mappings/net/minecraft/server/network/ServerPacketSender.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_5629 net/minecraft/server/network/ServerPacketSender
```

Comment:
Hmm, this isn't quite a sender. Imo this represents a packet recipient/consumer, or just a server-side representation of a client more

Reply:
Now that I look, this is more related to entity watching. This is like an audience for entity watching when you look at its usages